### PR TITLE
Implemented PNR divide

### PR DESCRIPTION
**Pull request by creduo**
This is an implementation of UniversalService/ProviderReservationDivideReq.

### CODE REVIEW

**Comment by creduo** [comment]
This PR may contain breaking changes as BookingTravelerName is added in the noCollapseList. Please review carefully regarding this change.

File: src/Services/Air/AirParser.js [hunk 1]
```diff
@@ -1084,6 +1084,29 @@ function universalRecordRetrieveRequest(data) {
   return response;
 }
 
+function formatProviderReservationInfo(data) {
+  return {
+    traveler: data['common_v47_0:BookingTravelerName'].map((travelerDetails) => {
```
Comment (dchertousov) [inline src/Services/Air/AirParser.js:1089]: Would prefer to call it passengers, as we never used `traveler` before and it is array.

File: test/Air/Air.test.js [hunk 2]
```diff
@@ -1740,6 +1740,26 @@ describe('#AirService', () => {
...
+      const createAirService = proxyquire('../../src/Services/Air/Air', {
+        './AirService': service,
+      });
+      createAirService({ auth }).providerReservationDivide({
+        uapi_ur_locator: 'AAABBB',
+        version: '1',
+        provider: '1G',
+        pnr: '456ABC',
+        passengers: [
+          { uapi_ref_key: 'XX/ADFH2345AGJDS245IGJSDifgjoai22343s==' },
```
Comment (dchertousov) [inline test/Air/Air.test.js:1756]: It's `uapi_passenger_ref` in example and templates. How could this test work?

Comment (creduo) [reply test/Air/Air.test.js]: renamed it to uapi_passenger_ref last minute, this must have been left out

File: docs/Air.md [hunk 3]
```diff
@@ -437,6 +437,23 @@ Gets pnr information and tickets list from [`importPNR`](#importPNR) and then do
...
+Sends a pnr divide request and returns parent and child reservation information with divided passenger list.
+
+**Returns**: `Promise` 
+
+**See**: [Dividing Universal Records](https://support.travelport.com/webhelp/uapi/uapi.htm#Booking/UniversalRecord/Dividing_Universal_Records.htm%3FTocPath%3DUniversal%2520Records%2520and%2520Bookings%7C_____5)
+and <a href="../examples/Air/divide.js">providerReservationDivide example</a>**
+
+| Param | Type | Description |
+| --- | --- | --- |
+| uapi_ur_locator | `Object` | Universal Record Locator |
```
Comment (dchertousov) [inline docs/Air.md:452]: It's not an object.
Also, could we allow to skip passing this locator as w we might get it internally?

Comment (dchertousov) [reply docs/Air.md:452]: And could you please add a description of the response?

**Review by dchertousov — CHANGES_REQUESTED** [review]
Thank you for that very helpful method!

Just few things are to be enhanced, and we are ready to roll with it

**Pull request closed without merging**